network/src/protocols/rpc/mod.rs

```diff
@@ -89,6 +89,11 @@ pub mod utils;
 #[cfg(test)]
 mod test;
 
+#[cfg(any(feature = "fuzzing", test))]
+#[path = "fuzz.rs"]
+/// fuzzing module for the rpc protocol
+pub mod fuzz;
```

Comment:
we use this pattern with other fuzzers, maybe we should try and use the same name for the fuzzing module. No preference as to which naming pattern is better :P

Reply:
do you mean like `event_processor_fuzzing.rs` vs `rpc/fuzz.rs`? would you prefer `rpc/fuzzing.rs` or `rpc/inbound_request_fuzzing.rs` or something else?